### PR TITLE
Add AAC depayloader

### PR DESCRIPTION
**Pull request by Janix4000**

### CODE REVIEW

File: test/membrane_mp4/demuxer/isom/integration_test.exs [hunk 1]
```diff
@@ -56,6 +56,39 @@ defmodule Membrane.MP4.Demuxer.ISOM.IntegrationTest do
...
+
+    structure =
+      child(:file, %Membrane.File.Source{location: in_path})
+      |> child({:parser, :in}, %Membrane.AAC.Parser{
+        in_encapsulation: :ADTS,
+        out_encapsulation: :none
+      })
+      |> child(:payloader, Membrane.MP4.Payloader.AAC)
+      |> child(:depayloader, Membrane.MP4.Depayloader.AAC)
+      |> child({:parser, :out}, %Membrane.AAC.Parser{
```
Comment (varsill) [inline test/membrane_mp4/demuxer/isom/integration_test.exs:72]: don't we want to do the same trick as with the h264 payloder - that is, prepare a bin, that would have a parser?

File: test/membrane_mp4/demuxer/isom/integration_test.exs [hunk 2]
```diff
@@ -56,6 +56,39 @@ defmodule Membrane.MP4.Demuxer.ISOM.IntegrationTest do
...
+
+    assert_end_of_stream(pipeline, :sink, :input, 6000)
+    refute_sink_buffer(pipeline, :sink, _buffer, 0)
+
+    assert :ok == Pipeline.terminate(pipeline, blocking?: true)
+
+    in_aac = File.read!(in_path)
+    out_aac = File.read!(out_path)
+
+    # there are some different single bytes in the depayloaded file
```
Comment (varsill) [inline test/membrane_mp4/demuxer/isom/integration_test.exs:88]: I believe we should inspect why the files are different

File: lib/membrane_mp4/depayloader/aac.ex [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  defp get_esds_content(esds) do
+    elementary_stream_id = 1
+    stream_priority = 0
+
+    {esds, <<>>} = unpack_esds_section(esds, 3)
+
+    <<^elementary_stream_id::16, ^stream_priority, rest::binary>> = esds
+
+    {section_4, esds_section_6} = unpack_esds_section(rest, 4)
+    {_section_6, <<>>} = unpack_esds_section(esds_section_6, 6)
```
Comment (varsill) [inline lib/membrane_mp4/depayloader/aac.ex:49]: As far as I know that should also be ok:
```suggestion
    {<<2>>, <<>>} = unpack_esds_section(esds_section_6, 6)
```

File: test/membrane_mp4/demuxer/isom/integration_test.exs (hunk 1 above)
Comment (Janix4000) [reply test/membrane_mp4/demuxer/isom/integration_test.exs:72]: In AAC.Payloader, the buffers remain unchanged, so there is no need to parse them afterward.

File: test/membrane_mp4/demuxer/isom/integration_test.exs (hunk 2 above)
Comment (Janix4000) [reply test/membrane_mp4/demuxer/isom/integration_test.exs]: Changed tests

**Review by varsill — APPROVED** [review]

**Pull request merged**